lib/server/services/qualityControlFlag/QcFlagService.js

```diff
@@ -450,6 +340,70 @@ class QcFlagService {
             .include({ association: 'verifications', include: [{ association: 'createdBy' }] })
             .groupBy('id');
     }
+
+    /**
+     * Find a run with a given run number and including a given detector, linked to a given data/simulation pass, and throw if none exists
```

Comment:
I would say: `Find a run with a given run number linked to given detector and a data/simulation pass`